python/python-core/src/main/python/domain/_annotations.py

```diff
@@ -653,13 +653,13 @@ def __init__(self, *,
 
 class DeepPlanningClone(JavaAnnotation):
     """
-    Marks a problem fact class as being required to be deep planning cloned.
```

Comment:
Please keep this in the docstring; attributes that are instances of a planning_entity and planning_solution type are automatically deep cloned.